helion/autotuner/base_search.py

```diff
@@ -188,6 +188,10 @@ def _compute_baseline(self) -> tuple[object, bool, Sequence[object] | None]:
                     baseline_config,
                     prefix=f"Generated Triton code for {decorator}:",
                 )
+                if self.settings.print_repro:
```

Comment:
Maybe move the config check into the helper so you don't need to check this so much?